bindata/cert-manager-deployment/controller/cert-manager-cluster-view-cr.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can you find out why this is being added exclusively, since we already have a cluster role under bindata.

Reply:
https://github.com/cert-manager/cert-manager/pull/6241 -- intended at adding specific rbac for certain cluster view users, and it should work with OpenShift users due to the annotation.